src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+
+    private String name = "Rectangle";
+    private double firstLeg, secondLeg;
```

Comment:
Consider using more descriptive names for the sides of the rectangle, such as 'width' and 'height' instead of 'firstLeg' and 'secondLeg'.